Products/zms/standard.py

```diff
@@ -2191,7 +2191,7 @@ def dt_exec(context, v, o={}):
   @rtype: C{any}
   """
   if type(v) is str:
-    if v.startswith('##'):
+    if v.startswith('## Script (Python)'):

```

Comment:
this would cause incompatibilities with existing code and should be avoided!

Reply:
this change is revoked: 
https://github.com/zms-publishing/ZMS/pull/139/commits/68915ba566c7668b0eaf7cf6b63358febbcad119
To avoid the misinterpreting I modified ZMSTextarea's standard-template:
https://github.com/zms-publishing/ZMS/pull/139/commits/568a5151c6c5364ee86c2d2cbe14591ebf778ee5

Now, in case of markdown the attr-processing provides only text:
![markdown_fix](https://user-images.githubusercontent.com/29705216/225078187-e2dc7b2d-5ec4-4042-a1f0-74608fc2b08e.gif)